Xamarin.Essentials/Share/Share.shared.cs

```diff
@@ -21,7 +24,16 @@ public static Task RequestAsync(ShareFileRequest request)
         }
     }
 
-    public class ShareTextRequest
+    public class ShareRequestBase
+    {
+        public string Title { get; set; }
+
+#if !NETSTANDARD1_0
+        public Rectangle SourceScreenPosition { get; set; } = Rectangle.Empty;
```

Comment:
I think "Position" implies "point". Maybe a Bounds or Frame. Or even Rectangle, but not too much a fan of this. I like Bounds the most.

Reply:
`PresentationSourceBounds`